Bump pkgconfig version to 2.0.2, from git commit 
8a2796be4b03ee6d4525b479ea4ebab970e1500c
--- a/SConstruct
+++ b/SConstruct
@@ -2,6 +2,10 @@
 import os
 import sys
 
+version="2.0.2"
+abi_major = 3
+abi_minor = 0
+
 cflags='-O2 -pipe -DHAVE_CONFIG_H '
 prefix='/usr/local'
 rpath=''
@@ -117,9 +121,6 @@
 libdatadir = libdir+'/sunpinyin/data'
 headersdir = prefix+'/include/sunpinyin-2.0'
 
-abi_major = 3
-abi_minor = 0
-
 #
 #==============================environment==============================
 #
@@ -326,7 +327,7 @@
         '',
         'Name: libsunpinyin',
         'Description: IME library based on Statistical Language Model',
-        'Version: 2.0.1',
+        'Version: %s' % version,
         'Requires: sqlite3',
         'Libs: -L${libdir} -lsunpinyin',
         'Cflags: ' + reduce(lambda a, b: a + ' ' + b,
